core-services/logging/fluentbit/opensearch_output.conf.tftpl

```diff
@@ -7,6 +7,7 @@
     HTTP_Passwd admin
     Logstash_Format True
     Logstash_Prefix k8s-pod-logs
+    Logstash_Prefix_Key kubernetes_container_name
```

Comment:
Looking through issues, I don't think we can use two values with `fluentbit` yet.  If we can only use one of `namespace` or `container_name`, I'm leaning toward namespace.  That prevents the issue of people poorly naming their containers (a dozen different `main` containers) - this could also just show up when a pod has an `nginx` sidecar that will get grouped with the ingress controller container.

Reply:
right. there's still a chance people don't use great namespace names / separation e.g. a ton of logs flowing through `default`. But I switched to namespace in latest commit and verified this filter is working in GKE.